editor/enable-tracking-prompt/test/index.js

```diff
@@ -0,0 +1,79 @@
+/**
+ * External dependencies
+ */
+import { mount } from 'enzyme';
```

Comment:
>`shallow` doesn't work here because we reach inside a `Button` element and grab its text.  Open to other suggestions.

Remarked at https://github.com/WordPress/gutenberg/pull/2140#discussion_r131301830. It wasn't entirely as simple, but close:

```diff
diff --git a/editor/enable-tracking-prompt/test/index.js b/editor/enable-tracking-prompt/test/index.js
index 686fa0ec..098b95e0 100644
--- a/editor/enable-tracking-prompt/test/index.js
+++ b/editor/enable-tracking-prompt/test/index.js
@@ -1,7 +1,7 @@
 /**
  * External dependencies
  */
-import { mount } from 'enzyme';
+import { shallow } from 'enzyme';
 
 /**
  * Internal dependencies
@@ -29,13 +29,13 @@ describe( 'EnableTrackingPrompt', () => {
 	} );
 
 	it( 'should render a prompt with Yes and No buttons', () => {
-		const prompt = mount(
+		const prompt = shallow(
 			<EnableTrackingPrompt />
 		);
-		const buttons = prompt.find( '.button' );
+		const buttons = prompt.find( 'Button' );
 		expect( buttons.length ).toBe( 2 );
-		expect( buttons.at( 0 ).text() ).toBe( 'Yes' );
-		expect( buttons.at( 1 ).text() ).toBe( 'No' );
+		expect( buttons.at( 0 ).children().text() ).toBe( 'Yes' );
+		expect( buttons.at( 1 ).children().text() ).toBe( 'No' );
 
 		expect( window.setUserSetting )
 			.not.toHaveBeenCalled();
@@ -46,11 +46,11 @@ describe( 'EnableTrackingPrompt', () => {
 	} );
 
 	it( 'should enable tracking when clicking Yes', () => {
-		const prompt = mount(
+		const prompt = shallow(
 			<EnableTrackingPrompt removeNotice={ removeNotice } />
 		);
-		const buttonYes = prompt.find( '.button' )
-			.filterWhere( node => node.text() === 'Yes' );
+		const buttonYes = prompt.find( 'Button' )
+			.filterWhere( node => node.children().text() === 'Yes' );
 		buttonYes.simulate( 'click' );
 
 		expect( window.setUserSetting )
@@ -62,11 +62,11 @@ describe( 'EnableTrackingPrompt', () => {
 	} );
 
 	it( 'should disable tracking when clicking No', () => {
-		const prompt = mount(
+		const prompt = shallow(
 			<EnableTrackingPrompt removeNotice={ removeNotice } />
 		);
-		const buttonNo = prompt.find( '.button' )
-			.filterWhere( node => node.text() === 'No' );
+		const buttonNo = prompt.find( 'Button' )
+			.filterWhere( node => node.children().text() === 'No' );
 		buttonNo.simulate( 'click' );
 
 		expect( window.setUserSetting )
```